uwblueprint/humane-society
# changed activity to activity type

## Notion ticket link

[Notion Ticket](https://www.notion.so/uwblueprintexecs/Rename-Activity-to-Activity-Type-11910f3fb1dc8041b3a7e9fa2030377f?pvs=4)



## Implementation description
* Changed all occurrences of activity to activityType



## Steps to test
1. Make sure migration up and down both work
2. Make sure all the endpoints still work



## What should reviewers focus on?
* 


## Checklist
